packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -1,10 +1,10 @@
-Dropdown containers are elements that display a menu with additional elements as soon as a user clicks on the top element, e.g. a dropdown button. Clicking on one of the items in the dropdown triggers the action. We typically use dropdowns to allow users to select one option from a list of available choices.
+Dropdown containers are elements that display a menu with additional items as soon as a user clicks on the top element, e.g. a dropdown button. Clicking on one of the items in the dropdown triggers the action. We typically use dropdowns to allow users to select one option from a list of available choices.
```

Comment:
Maybe you could make this sentence more simple, like "Dropdowns display a menu ..."